keyboards/jj4x4/README.md

```diff
@@ -16,6 +16,24 @@ Flashing example for this keyboard ([after setting up the bootloadHID flashing e
 
     make jj4x4:default:flash
 
-**Reset Key**: Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard.
+**Reset Key**:
 
-See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+Hold down the key located at *K00*, commonly programmed as *8* while plugging in the keyboard usb cable.
+
+Key *K00* is in the above picture:
+
+Row 3 from bottom (or 1 from top), Column 2 from left (or 3 from right), just like the schematic below:
```

Comment:
```suggestion
Row 2 from the top, Column 2 from the left, see the schematic below:
```
This is optional but makes more sense in my opinion

Reply:
sure, it's more simple.